osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs

```diff
@@ -500,7 +500,7 @@ public sealed override HitSampleInfo With(Optional<string> newName = default, Op
                 => new LegacyHitSampleInfo(newName.GetOr(Name), newBank.GetOr(Bank), newVolume.GetOr(Volume), newCustomSampleBank.GetOr(CustomSampleBank), newIsLayered.GetOr(IsLayered));
 
             public bool Equals(LegacyHitSampleInfo? other)
-                => base.Equals(other) && CustomSampleBank == other.CustomSampleBank && IsLayered == other.IsLayered;
+                => base.Equals(other) && CustomSampleBank == other.CustomSampleBank;
```

Comment:
`IsLayered` should also not be accounted in `GetHashCode` right? otherwise the dictionary lookup will still fail if I recall how it works correctly.

Reply:
Good call. It won't cause a failure, but may reduce the lookup efficiency I think.